src/appleseed/renderer/modeling/edf/edf.cpp

```diff
@@ -108,8 +108,6 @@ bool EDF::on_frame_begin(
             get_path().c_str());
     }
 
-    m_max_contribution = get_uncached_max_contribution();
```

Comment:
You can also remove the member `m_max_contribution`.

Reply:
It is already removed.